content/workflows/csp/aws/service/kms/resources/cmk/policy-context/backing-key-rotation-not-enabled.yaml

```
@@ -0,0 +1,29 @@
+schema: 1.0
+type: policy-context
+id: backing-key-rotation-not-enabled
```

```suggestion
id: cmk-rotation-not-enabled
```

Please rename file as well